lib/jquery-rjs/rendering.rb

```diff
@@ -1,13 +1,21 @@
 require 'action_view/helpers/rendering_helper'
 
-ActionView::Helpers::RenderingHelper.module_eval do
-  def render_with_update(options = {}, locals = {}, &block)
-    if options == :update
-      update_page(&block)
-    else
-      render_without_update(options, locals, &block)
-    end
+module JqueryRjs::RenderingHelper
+  method_name = "render#{'_with_update' if RUBY_VERSION < '2'}"
+  define_method(method_name) do |options = {}, locals = {}, &block|
+    update_page(&block) if options == :update
+
+    args = options, locals, block
+    RUBY_VERSION < '2' ? render_without_update(*args) : super(*args)
```

Comment:
> but this is technically changing the existing functionality

I don't see that. Where? (Note I had to fix some code with your first comment, should be the same semantics now though)

> What I am suggesting is "on load, if ruby 2, define this method, otherwise this method", instead of "when this method is called, are we still ruby 2?

That's fine, it just duplicates logic for the method. The way to do this is to just have two different modules and prepend/include and alias one depending on the ruby version.

```
module LegacyRenderingHelper
end

module RenderingHelper
end

if RUBY_VERSION < '2'
  # eval the Rails module, include the LegacyRenderingHelper, alias_method_chain
else
  # prepend the normal RenderingHelper
end
```

The issue with this is mostly the fact that I would have to either duplicate the entire module of helper methods (in the other patch in this PR) or do some Ruby module mincing of a third module before doing prepend/alias_method_chain. So...kind of a tossup. Thoughts on that?